### ISSUE TITLE
Ratification of Extended Measurement Or Facts extention

### ISSUE THREAD

**Issue by Daphnisd** [tdwg/dwc-qa#81]
At the GEO BON data mobilization symposium last week @kbraak and Dmitry Schigel let me know that GBIF would benefit from ratification by TDWG of the custom [eMoF extension ](http://rs.gbif.org/extension/obis/extended_measurement_or_fact.xml) created by OBIS and described at https://bdj.pensoft.net/articles.php?id=10989&instance_id=3385374.

I assume the problem is that extension offers a workaround the star schema. I'm confused as to whether the star schema is a GBIF implementation or a TDWG standard. 
@tucotuco, can you comment? 

What would be needed for GBIF to promote/allow the use of the eMoF? Is TDWG ratification needed, and if so, what exactly needs to be ratified?

**Comment by jmacklin** [comment]
Hi Daphnis,

John can say more but TDWG has not ratified DwC extensions in the past
however, we have just ratified a new set of procedures (Documentation and
Maintenance Standard) that will make this possible and there are a couple
of extensions that have been put forward for review. So, if one of the
authors of the Measurement of Fact extension wants to submit it to TDWG we
can put it into the review process. Extensions that are not "ratified" can
certainly still be used :-)

The star schema is an implementation used in the Integrated Publishing
Toolkit (IPT) developed by GBIF and is thus not related to TDWG.

Best,  JAmes

**Comment by tucotuco** [comment]
I second what James has said here. In practical terms of publication, the
star schema is a current limitation, the demise of which is likely
forthcoming with the advent of a W3C recommendation for file sharing of
related CSV files, which GBIF (Time Robertson) has been participating in
developing.

**Comment by dagendresen** [comment]
I agree that "ratification" of the eMoF extension might not be needed or useful. However, the eMoF extension introduces three new terms that are not (yet?) part of Darwin Core. Namely measurementTypeID, measurementValueID and measurementUnitID.

However, there already exists dwciri:measurementType and dwciri:measurementUnit (but not dwciri:measurementValue), http://rs.tdwg.org/dwc/terms/guides/rdf/index.htm
